### PR TITLE
add log files to deploy and run tasks

### PR DESCRIPTION
**Pull request by allendm-msft**
append the dta related logs onto build

### CODE REVIEW

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 [hunk 1]
```diff
@@ -328,10 +328,23 @@ function Set-TestAgentConfiguration
...
+    }
+
     DeleteDTAAgentExecutionService -ServiceName "DTAAgentExecutionService" | Out-Null
 
     $configOut = InvokeTestAgentConfigExe -Arguments $configArgs -Version $TestAgentVersion -UserCredential $MachineUserCredential
 
+    if(Test-path -Path $configLogFile) 
+    {
+        Write-Verbose "=== Starting to print the testagent configuration log file ==="
+        Get-Content $configLogFile | foreach { Write-Verbose $_ }
```
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: Get-Content $file | Write-Verbose. directly pipe the entire content

File: Tasks/RunDistributedTests/TestAgentUnRegistration.ps1 [hunk 2]
```diff
@@ -135,6 +135,13 @@ function TestAgent-UnRegister
         [String] $TestAgentVersion 
     )
 
+    $dtaLogFile = Join-Path $env:SystemDrive "DtaLogs" | Join-Path -ChildPath "DTAExecutionHost.exe.log"
+    if(Test-path -Path $dtaLogFile) 
+    {
+        Write-Verbose "=== Starting to print the dtaexecutionhost log file ==="
+        Get-Content $dtaLogFile | foreach { Write-Verbose $_ }
+        Write-Verbose "=== Done printing the dtaexecutionhost log file ==="
```
Comment (nigurr) [inline Tasks/RunDistributedTests/TestAgentUnRegistration.ps1]: Same here

**Comment by nigurr** [comment]
Add machine information [FQDN]? It would be hard to differentiate the log coming from which machine

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 [hunk 3]
```diff
@@ -328,10 +328,23 @@ function Set-TestAgentConfiguration
         }
     }    
 
+    $configLogFile = Join-Path $env:temp "testagentconfig.log"
+    if(Test-path -Path $configLogFile)
```
Comment (nigurr) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: We don't need to test the path if we are Silently Continuing.

File: Tasks/RunDistributedTests/TestAgentUnRegistration.ps1 [hunk 4]
```diff
@@ -135,6 +135,13 @@ function TestAgent-UnRegister
         [String] $TestAgentVersion 
     )
 
+    $dtaLogFile = Join-Path $env:SystemDrive "DtaLogs" | Join-Path -ChildPath "DTAExecutionHost.exe.log"
+    if(Test-path -Path $dtaLogFile) 
+    {
+        Write-Verbose "=== Starting to print the dtaexecutionhost log file ==="
+        Get-Content $dtaLogFile | foreach { Write-Verbose $_ }
```
Comment (nigurr) [inline Tasks/RunDistributedTests/TestAgentUnRegistration.ps1]: same here

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 (hunk 3 above)
Comment (allendm-msft) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: not needed.. removed

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 (hunk 1 above)
Comment (allendm-msft) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: using it to prefix the computer name per line

File: Tasks/RunDistributedTests/TestAgentUnRegistration.ps1 (hunk 4 above)
Comment (allendm-msft) [inline Tasks/RunDistributedTests/TestAgentUnRegistration.ps1]: sma

File: Tasks/RunDistributedTests/TestAgentUnRegistration.ps1 (hunk 2 above)
Comment (allendm-msft) [inline Tasks/RunDistributedTests/TestAgentUnRegistration.ps1]: added computername to all "printing" lines

**Comment by nigurr** [comment]
Looks good to me

**Pull request merged**